### PR TITLE
feat: Add operators data fetch at startup

### PR DESCRIPTION
**Pull request by JulianVentura**
### Motivation

We need to store some information about operators in order to provide better context on telemetry data. We also want to stop fetching the L1 smart contract on every operator version update.

### Description

This PR adds data fetch for all registered operators at telemetry service startup. This data includes the operator `address`, `id`, `stake` and `name` and it's stored on the already implemented database.

### CODE REVIEW

File: telemetry_api/.env.dev [hunk 1]
```diff
@@ -1 +1,2 @@
 export ALIGNED_CONFIG_FILE="../contracts/script/output/devnet/alignedlayer_deployment_output.json"
+export ENVIRONMENT="devnet"
```
Comment (JuArce) [inline telemetry_api/.env.dev]: It is not necessary to use export

Comment (JuArce) [inline telemetry_api/.env.dev]: It is not necessary to use quotes

File: telemetry_api/lib/telemetry_api/contract_managers/delegation_manager.ex [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  def get_contract_address() do
+    @contract_address
+  end
+
+  def get_operator_url(operator_address) do
+    DelegationManager.EventFilters.operator_metadata_uri_updated(operator_address)
+      |> Ethers.get_logs(fromBlock: @first_block)
+      |> case do
+        {:ok, data} -> 
+          url = List.last(data).data |> hd() # most recent entry
```
Comment (JuArce) [inline telemetry_api/lib/telemetry_api/contract_managers/delegation_manager.ex]:
```suggestion
          url = List.last(data).data |> hd() # The head (hd) is the most recent entry
```

File: telemetry_api/lib/telemetry_api/contract_managers/operator_state_retriever.ex [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  use Ethers.Contract,
+    abi_file: "priv/abi/OperatorStateRetriever.json",
+    default_address: @contract_address
+
+  def get_contract_address() do
+    @contract_address
+  end
+
+
+  def get_operators() do
```
Comment (JuArce) [inline telemetry_api/lib/telemetry_api/contract_managers/operator_state_retriever.ex]: This function seems a little messy

**Review by JuArce — CHANGES_REQUESTED** [review]

File: telemetry_api/lib/telemetry_api/contract_managers/operator_state_retriever.ex [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    with {:ok, block_number} = Ethers.current_block_number() do
+          quorum_numbers = <<0>>
+          response = __MODULE__.get_operator_state(@registry_coordinator_address, quorum_numbers, block_number) |> Ethers.call()
+          case response do
+            {:ok, [operators | _]} -> 
+              operators = 
+                operators |> Enum.map(fn op_data -> 
+                  {address, id, stake} = op_data
```
Comment (avilagaston9) [inline telemetry_api/lib/telemetry_api/contract_managers/operator_state_retriever.ex]: You can directly do something like this:
```Elixir
operators |> Enum.map(fn {address, id, stake} -> ...
```

File: telemetry_api/.env.dev (hunk 1 above)
Comment (JulianVentura) [reply telemetry_api/.env.dev]: Done

File: telemetry_api/lib/telemetry_api/contract_managers/delegation_manager.ex (hunk 2 above)
Comment (JulianVentura) [reply telemetry_api/lib/telemetry_api/contract_managers/delegation_manager.ex]: Done

File: telemetry_api/lib/telemetry_api/contract_managers/operator_state_retriever.ex (hunk 3 above)
Comment (JulianVentura) [reply telemetry_api/lib/telemetry_api/contract_managers/operator_state_retriever.ex]: Fixed

File: telemetry_api/lib/telemetry_api/contract_managers/operator_state_retriever.ex (hunk 4 above)
Comment (JulianVentura) [reply telemetry_api/lib/telemetry_api/contract_managers/operator_state_retriever.ex]: Cool! didn't know that. I have updated it

**Review by JuArce — APPROVED** [review]

**Pull request merged**